hibike/hibike_process.py

```diff
@@ -168,15 +168,16 @@ def helper():
                         ], 0.1)
                 elif hm.devices[hm.uid_to_device_id(uid)]["name"] == "YogiBear":
                     set_interval_sequence([
-                        make_send_write(pipeToChild, uid, [("enable", True),("duty_cycle", 0)]),
-                        make_send_write(pipeToChild, uid, [("enable", True),("duty_cycle", 0.5)]),
-                        make_send_write(pipeToChild, uid, [("enable", True),("duty_cycle", 1.0)]),
-                        make_send_write(pipeToChild, uid, [("enable", True),("duty_cycle", 0)]),
-                        make_send_write(pipeToChild, uid, [("enable", True),("duty_cycle", -0.5)]),
-                        make_send_write(pipeToChild, uid, [("enable", True),("duty_cycle", -1.0)]),
-                        make_send_write(pipeToChild, uid, [("enable", True),("duty_cycle", 0)])
-
-                        ], 0.75)
+                        ##This is the test to see if the motor drives correctly
+                        #make_send_write(pipeToChild, uid, [("duty_cycle", 0)]),
+                        #make_send_write(pipeToChild, uid, [("duty_cycle", -1.0)]),
```

Comment:
This is generic testing code.
I dont think your pull request wants to change it. 
Unless your changes render this test obsolete, in which case you should either 
1) rewrite the test to work in your new changes
2) delete the tests

Either way, we should not have large blocks of commented out code in master.